barman/cli.py

```diff
@@ -899,6 +899,18 @@ def rebuild_xlogdb(args):
             help="The name of the AWS region containing the EC2 VM and storage "
             "volumes for recovery of a snapshot backup",
         ),
+        argument(
+            # TODO: change name of flag after definition
+            "--local-staging-path",
+            dest="local_staging_path",
```

Comment:
```suggestion
```

`dest` is not needed in this case as by default `argparse` replaces `-` with `_` to build the variable in the namespace object, so the name would be equivalent.

(see https://github.com/python/cpython/blob/main/Lib/argparse.py#L1562-L1573)